signing_service/signing_service/signing_service.py

```diff
@@ -136,6 +144,11 @@ def _parse_arguments() -> argparse.Namespace:
         type=int,
         help='Initial delay between reconnections, doubles after each unsuccessful attempt and is reset after success.',
     )
+    parser.add_argument(
+        'concent_public_key',
+        type=str,
+        help="Concent's public key.",
```

Comment:
Adding this parameter should be a separate commit.

Reply:
:+1: done